cmd/osctl/cmd/gen.go

```diff
@@ -39,13 +41,13 @@ var caCmd = &cobra.Command{
 		if err != nil {
 			helpers.Fatalf("error generating CA: %s", err)
 		}
-		if err := ioutil.WriteFile(organization+".crt", ca.CrtPEM, 0400); err != nil {
+		if err := ioutil.WriteFile(organization+".crt", ca.CrtPEM, 0600); err != nil {
```

Comment:
Do something in this PR require us to have these be `0600` now?

Reply:
we can't overwrite them at `r--`, so when calling hack/dev/gen.sh again to regenerate the bits, it'd fail.